app/frontend/app/models/partner-deal.js

```diff
@@ -7,7 +9,22 @@ export default DS.Model.extend({
   dealUrl: DS.attr('string'),
   dealDescription: DS.attr('string'),
   redemptionInfo: DS.attr('string'),
-  recurring: DS.attr('boolean'),
+  recurring: DS.attr('number'),
   hasCodes: DS.attr('boolean'),
-  code: DS.attr('string')
+  code: DS.attr('string'),
+  claimedAt: DS.attr('date'),
+
+  isRecurring: Ember.computed.gt('recurring', 0),
+
+  canRedeemAgain: function() {
+    var date = new Date(this.get('claimedAt'));
```

Comment:
It's been awhile but looking at it, using `setSeconds` may have caused it to edit the property directly.